lib/fathead/sqlalchemy/output.txt

```diff
@@ -1,405 +1,405 @@
+interfaces.PropComparator.any_	A										<section class="prog__container"><pre><code>any_()</code></pre><p>\ninherited from the any_() method of ColumnOperatorsProduce a any_() clause against the\nparent object.\nNew in version 1.1.\n</p></section>	http://docs.sqlalchemy.org/en/latest/orm/internals.html#sqlalchemy.orm.interfaces.PropComparator.any_
+MultipleResultsFound	A										<section class="prog__container"><pre><code>exception sqlalchemy.orm.exc.MultipleResultsFound</code></pre><p>A single database result was required but more than one were found.</p></section>	http://docs.sqlalchemy.org/en/latest/orm/exceptions.html#sqlalchemy.orm.exc.MultipleResultsFound
+items	A										<section class="prog__container"><pre><code>items() &#8594; list of D's (key, value) pairs, as 2-tuples</code></pre><p>\ninherited from the items() method of dict</p></section>	http://docs.sqlalchemy.org/en/latest/orm/internals.html#sqlalchemy.orm.instrumentation.ClassManager.items
+attributes.QueryableAttribute.match	A										<section class="prog__container"><pre><code>match(other, **kwargs)</code></pre><p>\ninherited from the match() method of ColumnOperatorsImplements a database-specific &#8216;match&#8217; operator.match() attempts to resolve to\na MATCH-like function or operator provided by the backend.\nExamples include:\nPostgreSQL - renders x @@ to_tsquery(y)\nMySQL - renders MATCH (x) AGAINST (y IN BOOLEAN MODE)\nOracle - renders CONTAINS(x, y)\nother backends may provide special implementations.\nBackends without any special implementation will emit\nthe operator as &#8220;MATCH&#8221;.  This is compatible with SQlite, for\nexample.\n</p></section>	http://docs.sqlalchemy.org/en/latest/orm/internals.html#sqlalchemy.orm.attributes.QueryableAttribute.match
+InspectionAttr	A										<section class="prog__container"><pre><code
```

Comment:
As well, it looks like you're missing a newline and a code block in the first sentence. The result should look more like:

```
Bases: <code>sqlalchemy.event.base.Events</code>

Define events specific to mappings.
```

But we're showing:

```
Bases: sqlalchemy.event.base.EventsDefine events specific to mappings
```